rubygem-faraday_middleware/rubygem-faraday_middleware.spec

```diff
@@ -0,0 +1,81 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from faraday_middleware-0.10.0.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name faraday_middleware
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.10.0
+Release: 1%{?dist}
+Summary: Various middleware for Faraday
+Group: Development/Languages
+License: MIT
+URL: https://github.com/lostisland/faraday_middleware
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix}rubygem(faraday) < 0.10
+Requires: %{?scl_prefix}rubygem(faraday) >= 0.7.4
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildArch: noarch
+
+%description
+Various middleware for Faraday.
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install -n %{SOURCE0}
```

Comment:
Remove `-n %{SOURCE0}` here, as you've built a new .gem in the prep/build stages and using SOURCE0 skips that (useful) work by going straight back to the .gem you unpacked.